examples/advanced_diffusion_training/train_dreambooth_lora_sdxl_advanced.py

```diff
@@ -597,7 +663,7 @@ def parse_args(input_args=None):
     parser.add_argument("--prodigy_decouple", type=bool, default=True, help="Use AdamW style decoupled weight decay")
     parser.add_argument("--adam_weight_decay", type=float, default=1e-04, help="Weight decay to use for unet params")
     parser.add_argument(
-        "--adam_weight_decay_text_encoder", type=float, default=None, help="Weight decay to use for text_encoder"
+        "--adam_weight_decay_text_encoder", type=float, default=1e-03, help="Weight decay to use for text_encoder"
```

Comment:
Is this not slightly backwards harming? Earlier, if folks would have used the script, the `adam_weight_decay_text_encoder` was always None, so it didn't have any effect. Now, if they continue to resume from that trained checkpoint, things will be different, no? 

So, with that said, I think it's best to not change it. 

WDYT?